### PR TITLE
Clear clinical submission resolver

### PR DESCRIPTION
**Pull request by UmmulkiramR**
has the clearClinicalSubmission resolver
also has auth added to make user information available in the query.

### CODE REVIEW

File: src/server.ts [hunk 1]
```diff
@@ -29,11 +31,16 @@ import { Server } from 'http';
...
 
-import { ApolloServer } from '@apollo/server';
-import { startStandaloneServer } from '@apollo/server/standalone';
+import { ApolloServer, ContextFunction } from '@apollo/server';
+import {
+  StandaloneServerContextFunctionArgument,
+  startStandaloneServer,
+} from '@apollo/server/standalone';
 import schema from './schemas/index';
+// import { EgoJwtData } from '@icgc-argo/ego-token-utils/dist/common';
```
Comment (demariadaniel) [inline src/server.ts:43]: Can remove commented code?
I would include EgoJwtData with the GlobalGqlContext Type, which should have `userJwtData: EgoJwtData | null;`

File: src/schemas/clinical-resolvers/clearClinicalSubmissionResolver.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    obj: unknown,
+    args: { programShortName: string; fileType: string; version: string },
+    contextValue: any,
+  ) => {
+    const { programShortName, fileType, version } = args;
+    const response = await submissionAPI.clearFileDataFromActiveSubmission(
+      programShortName,
+      fileType,
+      version,
+      (<GlobalGqlContext>contextValue).egoToken,
```
Comment (demariadaniel) [inline src/schemas/clinical-resolvers/clearClinicalSubmissionResolver.ts:35]: For consistency w/ other functions I would change line 28 to `contextValue: GlobalGqlContext,` and 
line 35 to `contextValue.egoToken,`

Comment (UmmulkiramR) [reply src/schemas/clinical-resolvers/clearClinicalSubmissionResolver.ts:35]: unfortunately, the _buildSubgraphSchema_ method does not accept resolvers of custom context type. Hence the typecast.

File: src/server.ts (hunk 1 above)
Comment (UmmulkiramR) [reply src/server.ts:43]: not sure why but argo clinical running locally is not able to fetch the type EgoJwtData from 
@icgc-argo/ego-token-utils/dist/common.

Comment (demariadaniel) [reply src/server.ts:43]: `@icgc-argo/ego-token-utils` needs to be installed

File: src/schemas/clinical-resolvers/clearClinicalSubmissionResolver.ts (hunk 2 above)
Comment (demariadaniel) [reply src/schemas/clinical-resolvers/clearClinicalSubmissionResolver.ts:35]: OK I see now, sorry VSCode TS version was not set correctly

File: src/server.ts (hunk 1 above)
Comment (UmmulkiramR) [reply src/server.ts]: Oh yes! Done. Thanks.

**Review by demariadaniel — APPROVED** [review]

**Pull request merged**